RadixWallet/Features/ImportMnemonic/VerifyMnemonic/VerifyMnemonic+View.swift

```diff
@@ -0,0 +1,154 @@
+import SwiftUI
+
+extension VerifyMnemonic.State {
+	public var viewState: VerifyMnemonic.ViewState {
+		let enumeratedWords = mnemonic.words.identifiablyEnumerated()
+		let wordViewStates = enumeratedWords.map { word in
+			let isConfirmationWord = wordsToConfirm.contains(word)
+			return VerifyMnemonic.ViewState.WordViewState(
+				word: word,
+				placeholder: isConfirmationWord ? "" : "•••••",
```

Comment:
See Slack, we agreed to use 4, not 5, dots, matching the BIP39 specification that all words (in English word list) are unambiguously identified by the first 4 chars.